internal/test/framework/framework.go

```diff
@@ -236,15 +242,44 @@ func (f *TestFramework) getClusterVersion() error {
 	return nil
 }
 
-// getLatestGithubRelease gets the latest github release for the wmcb repo. This release is specific to the cluster
-// version
-func (f *TestFramework) getLatestGithubRelease() error {
-	client := github.NewClient(nil)
+// getLatestCniPluginsVersion returns the latest 0.8.x version of CNI plugins in 'v<major>.<minor>.<patch>' format
+func (f *TestFramework) getLatestCniPluginsVersion() error {
+	releases, err := f.GetGithubReleases("containernetworking", "plugins")
+	if err != nil {
+		return err
+	}
+	// Iterating over releases to fetch versions from tag names
+	var cniPluginsVersions = []string{}
+	for _, release := range releases {
+		cniPluginsVersions = append(cniPluginsVersions, release.GetTagName())
+	}
+	// Sorting versions in reverse order so as to get latest version first
+	sort.Sort(sort.Reverse(sort.StringSlice(cniPluginsVersions)))
+	// Iterating over versions to find first 0.8.x version which is not a release candidate
+	for _, cniPluginsVersion := range cniPluginsVersions {
+		if strings.HasPrefix(cniPluginsVersion, "v0.8.") && !strings.Contains(cniPluginsVersion, "rc") {
+			f.LatestCniPluginsVersion = cniPluginsVersion
+			return nil
+		}
+	}
+	return fmt.Errorf("could not fetch latest 0.8.x version of CNI Plugins")
+}
 
+// GetGithubReleases gets all the github releases for a given owner and repo
+func (f *TestFramework) GetGithubReleases(owner string, repo string) ([]*github.RepositoryRelease, error) {
```

Comment:
Why is this method public? I see it being used in framework only..

Reply:
Made it private